src/index.html

```diff
@@ -1,22 +1,412 @@
+              <h3 class="category__title">Accessories</h3>
+            </article>
+
+            <div class="categories__button">
+              <a
+                class="categories__button__link categories__button__text"
+                href="#"
+              >
+                View all
+              </a>
+            </div>
+          </div>
+        </section>
+      </div>
+
+      <section id="main__image">
+        <img
+          src="images/image.png"
+          alt="background image"
+          class="main__image"
+        />
+      </section>
+
+      <div class="main__content container">
+        <section
+          class="about-us"
+          id="about-us"
+        >
+          <h2 class="section-title about-us__title">About us</h2>
+          <div class="about-us__content">
+            <div class="about-us__content__description">
+              <p>
+                We're a London-based tech company on a mission to remove
+                barriers between people and technology.
+              </p>
+              <p>
+                To do this, we're crafting intuitive, flawlessly connected
+                products that improve our lives without getting in the way.
+                <br />
+                No confusing tech-speak. No silly product names. Just artistry,
+                passion and trust. And products we're proud to share with our
+                friends and family. Simple.
+              </p>
+              <p>Because beautiful tech shouldn't be complicated.</p>
+            </div>
+          </div>
+        </section>
+
+        <section
+          class="contact"
+          id="contact"
+        >
+          <h2 class="section-title">Contact us</h2>
+          <div class="contacts__wrapper">
+            <form
+              onsubmit="this.reset(); return false;"
+              class="contact__form"
+              action="#"
+            >
+              <input
+                type="text"
+                name="Name"
+                placeholder="Name"
+                class="contact__form--field"
+                required
+                pattern="\w+"
+              />
+              <input
+                type="email"
+                name="Email"
+                placeholder="Email"
+                class="contact__form--field"
+                required
+              />
+              <textarea
+                name="Message"
+                placeholder="Message"
+                class="contact__form--field contact__form--messege"
+                required
+              ></textarea>
+              <button class="contact__button">Send</button>
+            </form>
+            <div class="contact__info">
+              <h4 class="contact__info__title">Phone</h4>
+              <a
+                href="#"
+                class="contact__info__text"
+              >
+                +1 234 5555-55-55
+              </a>
+              <h4 class="contact__info__title">Email</h4>
+              <a
```

Comment:
Missing email link format

https://www.freecodecamp.org/news/mailto-link-how-to-make-an-html-email-link-example-code/